src/App.js

```diff
@@ -3,31 +3,51 @@ import movieData from './data';
 // import logo from './logo.svg';
 import './App.css';
 import MovieContainer from './MovieContainer';
+import MovieDescription from './MovieDescription';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: []
-
+      movies: [],
+      clickedMovie: '',
+      clickedHome: ''
     }
   }
   componentDidMount = () => {
     this.setState({movies: movieData.movies})
-    console.log(movieData.movies)
   }
 
+  getClickedMovie = (id) => {
+    const numberId = parseInt(id)
+    const selectedMovie = this.state.movies.find(movie => movie.id === numberId)
+    this.setState({movies: [...this.state.movies], clickedMovie: selectedMovie})
+}
+
+  returnToMainView = () => {
+    this.setState({movies: [...this.state.movies], clickedMovie: '', clickedHome: ''})
+  }
 
   render() {
-    return (
-      <main className='App'>
-        <h1>Rancid Tomatillos</h1>
-        <MovieContainer movies={this.state.movies}/>
-      </main>
-    )
+    if(this.state.clickedMovie || this.state.clickedHome) {
+      return (
+        <main className='App'>
+          <h1>Rancid Tomatillos</h1>
+          <MovieDescription movies={this.state.clickedMovie} returnToMainView={this.returnToMainView}/>
+        </main>
+      )
+    } else {
+      return (
+        <main className='App'>
+          <h1>Rancid Tomatillos</h1>
+          <MovieContainer movies={this.state.movies} getClickedMovie={this.getClickedMovie}/>
+        </main>
```

Comment:
Since you're showing the `main` and `h1` in both scenarios, what if the conditional rendering only handled the `MoveDescription` vs `MovieContainer` logic?